### ISSUE TITLE
[BUG] Many words can't be used on a sentence when trying to talk to the LLM

### ISSUE THREAD

**Issue by MicaelJarniac** [kercre123/wire-pod#363]
**Describe the bug**
Trying to ask the LLM for information about a game seems to always get Vector to start playing Blackjack.

Many other words have similar behavior, like "bad" and "start".

https://github.com/kercre123/wire-pod/blob/daf374cc9febd6c2ea8c70146eed58035a85aef2/chipper/intent-data/en-US.json

**To Reproduce**
1. Say "Hey Vector, what can you tell me about the Minecraft game?"
2. Vector starts Blackjack

**Expected behavior**
The prompt shouldn't match any known intents, thus forwarding it to the LLM.

**Screenshots**

**Device you are running wire-pod on (please complete the following information):**
 - OS: Ubuntu
 - Hardware: Dell Laptop

**Additional context**
```
2024.07.08 20:56:55: Intent matched: intent_play_blackjack, transcribed text: 'what can you talk about the minecraft game', device: 00706f36
2024.07.08 20:57:07: Intent matched: intent_play_blackjack, transcribed text: 'what can you tell me about the mind curve game', device: 00706f36
```

**Comment by MicaelJarniac** [comment]
A possible solution would be to have something similar to "I have a question", but for talking directly to the LLM, thus skipping intent matching.

**Comment by Zomboy4313** [comment]
WirePod is intent matching since you said 'game', saying I have a question is the method to always send responses to the LLM

**Comment by MicaelJarniac** [comment]
Solved by #368.
